plugins/base/src/main/java/edu/wpi/first/shuffleboard/plugin/base/BasePlugin.java

```diff
@@ -202,4 +243,15 @@
     );
   }
 
+  @Override
+  public List<Group> getSettings() {
+    return ImmutableList.of(
+            Group.of("Graph settings",
+                    Setting.of("Graph Update Rate",
+                            "How many hz (times a second) graph widgets update at. 25hz is considered the maximum tested",
+                            graphUpdateRate
+                    )
```

Comment:
Be clear and avoid parentheticals in setting summaries:

```suggestion
                    Setting.of(
                      "Graph Update Rate",
                      "How many times a second graph widgets update at. Faster update rates may cause performance issues",
                      graphUpdateRate
                    )
```